packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -4,9 +4,8 @@ import PropTypes from 'react-proptypes'
 import { withMutations } from 'cozy-client'
 import { translate } from 'cozy-ui/react/I18n'
 
```

Comment:
This commit (73178ab190a997dc851a5ac36d11a341d36c9d35) is a bit hard to follow and I am not sure I can properly review it.